include/boost/geometry/util/range.hpp

```diff
@@ -116,10 +116,10 @@ front(Range const& rng)
 
 /*!
 \brief Short utility to conveniently return the front element of a Range.
-\ingroup utility
-*/
+ \ingroup utility
+ */
```

Comment:
OK, I have one git question: shall I amend the commit (like I just did to adjust the comment) or make a new commit with the removed spaces ?